components/fieldgroups/intent/intent-details.schema.json

```diff
@@ -0,0 +1,107 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/intent-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Intent Details",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/intent"],
+  "description": "Captures and maintains additional intent attributes.",
+  "definitions": {
+    "intent-details": {
+      "properties": {
+        "xdm:topicID": {
+          "title": "Topic ID",
+          "description": "Unique ID for Topic.",
+          "type": "string"
+        },
+        "xdm:categoryName": {
+          "title": "Category Name",
+          "description": "Name of the Category, Keyword set to which topic belongs.",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/b2b-source",
+          "type": "string"
+        },
+        "xdm:keywordSetId": {
+          "title": "Keyword Set ID",
+          "description": "Unique ID of the keyword set.",
+          "type": "string"
+        },
+        "xdm:scoreString": {
+          "title": "Score String",
+          "description": "Intent Score (High, Med, Low).",
+          "type": "string"
+        },
+        "xdm:scoreNumber": {
+          "title": "Score Number",
+          "description": "Intent Score numeric.",
+          "type": "number"
+        },
+        "xdm:scoreDelta": {
+          "title": "Score Delta",
+          "description": "Delta score from previous week.",
+          "type": "string"
+        },
+        "xdm:topSignalLocations": {
+          "title": "Top Signal Locations",
+          "description": "Array of locations where intent is shown.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "xdm:isDeleted": {
+          "title": "Deleted Flag",
+          "description": "Flag indicating if the record is deleted.",
+          "type": "boolean"
+        },
+        "xdm:recordeDate": {
```

Comment:
I think there is a typo in the field name; should it be `recordedDate`?

Reply:
Agreed.